lms/templates/dashboard.html

```diff
@@ -61,9 +61,13 @@
     });
   </%static:require_module>
   <%static:require_module module_name="js/learner_dashboard/entitlement_unenrollment_factory" class_name="EntitlementUnenrollmentFactory">
-    EntitlementUnenrollmentFactory({
-      dashboardPath: "${reverse('dashboard') | n, js_escaped_string}",
-      signInPath: "${reverse('signin_user') | n, js_escaped_string}"
+    ## Wait until the document is fully loaded before initializing the EntitlementUnenrollmentView
+    ## to ensure events are setup correctly.
+    $(document).ready(function() {
```

Comment:
We need to wait until the document has been fully loaded to initialize the `EntitlementUnenrollmentFactory` as the events that are applied by `accessible_modal.js` need to be applied AFTER the events that are applied by `toggle_login_modal.js`, which is loaded via the JS_PIPELINE.